linux_os/guide/system/logging/package_rsyslog-module-gtls_installed/rule.yml

```diff
@@ -0,0 +1,39 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Ensure rsyslog-module-gtls is installed'
+
+description: |-
+    TLS protocol support for rsyslog is installed.
+    {{{ describe_package_install(package="rsyslog-module-gtls") }}}
+
+rationale: |-
+    The rsyslog-module-gtls package provides Transport Layer Security (TLS) support
+    for the rsyslog daemon, which enables secure remote logging.
+
+severity: medium
+
+identifiers:
+    cce@sle12: CCE-91512-4
+    cce@sle15: CCE-91199-0
```

Comment:
I think you should assign new CCEs instead of moving the old CCEs from existing rules, that can be confusing for a lot of people that a CCE now means a different configuration.

Reply:
Yes. I also mentioned this in the other rule.

Reply:
Obsolete sles exclusive rule removed.